pkg/experiment/ingester/segment_test.go

```diff
@@ -445,6 +447,46 @@ func TestDLQRecovery(t *testing.T) {
 	sw.queryInputs(clients, inputs)
 }
 
+func TestUnsymbolizedLabelIsSet(t *testing.T) {
```

Comment:
It looks like this new test does not catch the panics we had.

I did the following test:
```diff
~/p/pyroscope (marcsanmi/add-unsymbolized-label) > git diff
diff --git a/pkg/experiment/ingester/segment.go b/pkg/experiment/ingester/segment.go
index ba864d9f3..38fe09bf3 100644
--- a/pkg/experiment/ingester/segment.go
+++ b/pkg/experiment/ingester/segment.go
@@ -544,7 +544,7 @@ func (v *sampleAppender) VisitSampleSeries(labels []*typesv1.LabelPair, samples
        }
        var n profilev1.Profile
        v.exporter.ExportSamples(&n, samples)
-       v.head.Ingest(&n, v.id, labels, v.annotations)
+       v.head.Ingest(v.profile, v.id, labels, v.annotations)
 }
```

I believe the bug is triggered by a pprof with sample labels.

Reply:
Updated the test to trigger the panic when changed like this in commit e7381020fe11e0c41084e2089a576c0b0779f135